sdk_container/src/third_party/coreos-overlay/profiles/coreos/base/package.mask

```diff
@@ -20,3 +20,8 @@
 # Python 3.12 is in portage-stable (currently testing), so avoid picking it
 # up. Update this to mask later versions when we switch to 3.11.
 >=dev-lang/python-3.12
+
+# Do not update to openldap 2.6.3+, to take 2 different steps of updating
+# openldap, 1) from 2.4 to 2.5, 2) do an Alpha release around 2023-08, and
+# 3) finally update from 2.5 to 2.6.
+>=net-nds/openldap-2.6.3
```

Comment:
Normally I'd suggest the following, but I don't think Gentoo is going to reintroduce 2.6.1.

```suggestion
>=net-nds/openldap-2.6
```

Reply:
Done